Application/Api/Education/Certificate/Generator/Repository/ESBD/EsbdGsJa.php

```diff
@@ -0,0 +1,80 @@
+<?php
+namespace SPHERE\Application\Api\Education\Certificate\Generator\Repository\ESBD;
+
+use SPHERE\Application\Education\Certificate\Generator\Repository\Element;
+use SPHERE\Application\Education\Certificate\Generator\Repository\Page;
+use SPHERE\Application\Education\Certificate\Generator\Repository\Section;
+use SPHERE\Application\Education\Certificate\Generator\Repository\Slice;
+use SPHERE\Application\People\Person\Service\Entity\TblPerson;
+
+/**
+ * Class EsbdGsJa
+ *
+ * @package SPHERE\Application\Api\Education\Certificate\Certificate\Repository\ESBD
+ */
+class EsbdGsJa extends EsbdStyle
+{
+
+    /**
+     * @return array
+     */
+    public function selectValuesTransfer()
+    {
+        return array(
+            1 => "wird versetzt",
+            2 => "wird nicht versetzt"
+        );
+    }
+
+    /**
+     * @param TblPerson|null $tblPerson
+     * @return Page
+     * @internal param bool $IsSample
+     *
+     */
+    public function buildPages(TblPerson $tblPerson = null){
```

Comment:
Codacy found an issue: [Opening brace should be on a new line](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)